devenv/docker/blocks/openldap-mac/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Fork of https://github.com/dinkel/docker-openldap
+
+FROM osixia/openldap
+
+LABEL maintainer="Grafana team <hello@grafana.com>"
+
+ENV OPENLDAP_VERSION 2.4.57
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
+        slapd=${OPENLDAP_VERSION}* \
+        ldap-utils && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN mv /etc/ldap /etc/ldap.dist
+
+VOLUME ["/etc/ldap", "/var/lib/ldap"]
+
+COPY ldap.conf /etc/ldap.dist/ldap.conf
+
+COPY modules/ /etc/ldap.dist/modules
+COPY prepopulate/ /etc/ldap.dist/prepopulate
+
+COPY entrypoint.sh /entrypoint.sh
+COPY prepopulate.sh /prepopulate.sh
```

Comment:
guessing this can't be done as mounts on the docker-compose?

Context: I would prefer to avoid having an image to build if possible

Reply:
I have skimmed the Docker block by configuring the `osixia/openldap` image.